docs/getting_started/requirements.md

```diff
@@ -156,41 +149,16 @@ Please note that the user interface might not look or behave exactly the same ac
 
     ### Supported eZ Platform Cloud setup
 
-    Because of the large range of possible configurations of eZ Publish legacy, there is no ready-made recommended setup.
+    Because of the large range of possible configurations of eZ Platform, there are many possiblities beyond what is provided in default recommended configuration.
```

Comment:
```suggestion
    Because of the large range of possible configurations of eZ Platform, there are many possiblities beyond what is provided in the default recommended configuration.
```